.github/workflows/scheduled-tests.yml

```diff
@@ -1,13 +1,12 @@
 name: "Scheduled tests"
-run-name: "Scheduled tests ${{ github.event_name != 'schedule' && format('- {0}', github.actor) }}"
+run-name: "Scheduled tests ${{ (github.event_name != 'schedule' && format('- {0}', github.actor)) || '' }}"
```

Comment:
Without the || '' it outputs false.

![Screenshot 2025-05-13 at 11 31 19 AM](https://github.com/user-attachments/assets/a8df0b6f-2529-474d-af03-3a5bbcd2f89e)